host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -183,7 +183,20 @@ defmodule HostCore.Actors.ActorModule do
       instance_id = Agent.get(agent, fn content -> content.instance_id end)
       Tracer.set_attribute("instance_id", instance_id)
 
-      {:ok, module} = Wasmex.Module.compile(bytes)
+      {:ok, stdin} = Wasmex.Pipe.new()
```

Comment:
👋 hey :) Thanks again for letting me know of your PR Bailey! I just want to let you know that with wasmex 0.8 one needs to set the read/write position of pipes to read them after they have been written to (e.g. to read stdout after the WASM program wrote to it, do `Wasmex.Pipe.seek(stdout, 0)`).

I don't see any use of pipes in this diff apart from creating and feeding them into wasi options, but wanted to let you know just in case :)

Reply:
ohhh, I just saw that TODO further down `in the future, poll these so we can forward the err/out pipes to our logger` so probably this is no problem at all for you :)

Reply:
Yes, we definitely have more work to do for full wasi support 😅